<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatInfo</class>
 <widget class="QWidget" name="ChatInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>458</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {background: #094747;}
QLineEdit {background-color : #307070; border-width: 1px; border-radius: 0px; padding: 6px;}
QPushButton {background-color : #307070;border-width: 2px; border-radius: 0px; padding: 10px;}
QPushButton:pressed { background-color: #307777; }
QLabel {background: #0F5555; border-radius : 2px; border-width: 2px; padding: 6px; font: 18pt;}
QListWidget {background-color : #076060; border-radius : 0px; border-width: 2px; padding: 6px;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="userNameLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name Surname&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>userNameLabel</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="encrOptions">
     <property name="styleSheet">
      <string notr="true">QComboBox::drop-down {background: #699E93; width: 20px; border-width: 1px; border-radius: 6px; padding: 0px;}
QComboBox, QComboBox:on {border-width: 1px; border-radius: 6px; padding: 1px 1px 1px 6px;}
QComboBox:!editable {background: #3C867C;}
QComboBox:editable { background: #4C968C;}
QComboBox QAbstractItemView { border: 2px #DDDDDD; border-radius: 6px; background-color: #3B726B;}</string>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="encrLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {background: #094747; border-radius : 2px; border-width: 2px; padding: 6px; font: 14pt}
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;change encryption type:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLineEdit" name="addMemLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QListWidget" name="memList"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="addMemButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="memLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {background: #076060; font : 12pt}</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;members&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="horizontalSpacing">
      <number>1</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="changeNameButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {background: #156060; border-width: 2px; border-style: solid; border-radius: 5px; padding: 6px;}
QPushButton:pressed { background-color: #206767; }</string>
       </property>
       <property name="text">
        <string>change name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="changeSnameButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {background: #156060; border-width: 2px; border-style: solid; border-radius: 5px; padding: 6px;}
QPushButton:pressed { background-color: #206767; }</string>
       </property>
       <property name="text">
        <string>change surname</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="changeLogButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {background: #156060; border-width: 2px; border-style: solid; border-radius: 5px; padding: 6px;}
QPushButton:pressed { background-color: #206767; }</string>
       </property>
       <property name="text">
        <string>change login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
